com.unity.render-pipelines.universal/Runtime/ScriptableRenderer.cs

```diff
@@ -66,18 +66,15 @@ public static void SetCameraMatrices(CommandBuffer cmd, ref CameraData cameraDat
             // for now using cmd.SetViewProjecionMatrices
             //SetViewAndProjectionMatrices(cmd, viewMatrix, cameraData.GetDeviceProjectionMatrix(), setInverseMatrices);
             cmd.SetViewProjectionMatrices(viewMatrix, projectionMatrix);
-
-            // unity_MatrixInvVP is not set by cmd.SetViewProjectionMatrices, we set it here
-            Matrix4x4 viewAndProjectionMatrix = projectionMatrix * viewMatrix;
-            Matrix4x4 inverseViewProjection = Matrix4x4.Inverse(viewAndProjectionMatrix);
-            cmd.SetGlobalMatrix(ShaderPropertyId.inverseViewAndProjectionMatrix, inverseViewProjection);
-
-            cmd.SetGlobalMatrix(ShaderPropertyId.worldToCameraMatrix, viewMatrix);
```

Comment:
Note: This removal is intentional. We already set worldToCameraMatrix in the cmd.SetViewProjectionMatrices above, this is duplicated for now. We plan to remove cmd.SetViewProjectionMatrices and set all matrices explicitly in the future.